build.gradle.kts

```diff
@@ -107,3 +112,10 @@ publishing {
         }
     }
 }
+
+
+
+tasks.register<Test>("allTests") {
+    systemProperty("TESTS_WITH_EXTERNAL_SERVICES", "true")
+    useJUnitPlatform()
+}
```

Comment:
Please add a `test-all` target to the `Makefile` and update CI to call `make test-all` vs `make test`